coffee-rest/src/main/java/hu/icellmobilsoft/coffee/rest/validation/xml/JaxbTool.java

```diff
@@ -279,27 +282,82 @@ public String marshalXML(Object obj, String schemaPath) throws BaseException {
         return marshalXML(obj, schemaPath, properties);
     }
 
+    /**
+     * Marshals given XML {@link Object} to {@link String}. <br>
+     * Sets the following fix parameters for the conversion:
+     * <ul>
+     * <li>jaxb.formatted.output - TRUE ({@link Marshaller#JAXB_FORMATTED_OUTPUT})</li>
+     * <li>jaxb.fragment - TRUE ({@link Marshaller#JAXB_FRAGMENT})</li>
+     * </ul>
+     *
+     * @param obj
+     *            XML {@code Object}
+     * @param schemaPath
+     *            path to XSD or catalog to validate on, if null, then validation is not executed
+     * @return XML String
+     * @param additionalClasses
+     *            these classes will be added to the {@link JAXBContext}
+     * @throws BaseException
+     *             if invalid input or cannot be marshalled
+     */
+    public String marshalXML(Object obj, String schemaPath, Class<?>... additionalClasses) throws BaseException {
+        Map<String, Object> properties = new HashMap<>();
```

Comment:
latom mashol is levan hagyva az input check, probaljunk figyelni ra azert legalabb az uj methodoknal

Reply:
szerintem azert, mert mindegyik esetben le van kezelve a null vizsgalat a metoduson belul